src/components/ListItem.jsx

```diff
@@ -56,6 +76,10 @@ export function ListItem({ item, listPath }) {
 		}
 	}
 
+	useEffect(() => {
```

Comment:
I put a comment about this useEffect in the other PR but there is a way to do this without useEffect or setUrgency